plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -298,21 +348,19 @@ static void ffmpeg_source_tick(void *data, float seconds)
 			mp_media_free(&s->media);
 			s->media_valid = false;
 		}
-		s->destroy_media = false;
-	}
-}
 
-static void ffmpeg_source_start(struct ffmpeg_source *s)
-{
-	if (!s->media_valid)
-		ffmpeg_source_open(s);
+		s->destroy_media = false;
 
-	if (s->media_valid) {
-		mp_media_play(&s->media, s->is_looping);
-		if (s->is_local_file)
-			obs_source_show_preloaded_video(s->source);
-		set_media_state(s, OBS_MEDIA_STATE_PLAYING);
-		obs_source_media_started(s->source);
+		if (!s->is_local_file) {
+			FF_BLOG(LOG_ERROR, "thread has been destroyed");
```

Comment:
```suggestion
			FF_BLOG(LOG_ERROR, "reconnect thread has been destroyed");
```

Reply:
This line will likely trigger clang-format to fail for being too long.